### ISSUE TITLE
Set terser options

### ISSUE THREAD

**Issue by andrewtimberlake** [embroider-build/embroider#1353]
Because embroider is not using ember-cli-terser, I’m not sure how to set the `drop_console` option.
Before embroider I had the following in ember-cli-build.js
```
'ember-cli-terser': {
  terser: {
    compress: {
      drop_console: true,
    },
  },
},
```
Is it possible to set that option via embroider?

**Comment by Windvis** [comment]
Embroider uses [hardcoded options for terser](https://github.com/embroider-build/embroider/blob/b869b2c4d52bddaa347faba404cd283a7bd33493/packages/webpack/src/ember-webpack.ts#L335) so I don't think it's possible to configure terser at the moment.

I assume we could make it possible by adding a new `terserOptions` / `minifierOptions` object to the available [Webpack options](https://github.com/embroider-build/embroider/blob/main/packages/webpack/src/options.ts) which would then be merged with the Embroider terser options before being passed into terser?

Which would then look like this in apps:
```js
return require('@embroider/compat').compatBuild(app, Webpack, {
  packagerOptions: {
    terserOptions: {
      compress: {
        drop_console: true,
      }
    }
  }
});
```

**Comment by ef4** [comment]
That is correct but also it's important to note that our direct use of terser is only for the legacy scripts (so `vendor.js`).

All the rest of your app and addons goes through webpack's magnification, so you can adjust all the minification options by passing them in the webpack config.

**Comment by andrewtimberlake** [comment]
Thanks, that worked. I did the following:
```javascript
  const TerserPlugin = require('terser-webpack-plugin');
  //…
  const { Webpack } = require('@embroider/webpack');
  return require('@embroider/compat').compatBuild(app, Webpack, {
    packagerOptions: {
      webpackConfig: {
        optimization: {
          minimize: true,
          minimizer: [
            new TerserPlugin({
              terserOptions: {
                compress: {
                  drop_console: true,
                },
              },
            }),
          ],
        },
      },
    },
  });
```
